daffodil-test/src/test/resources/org/apache/daffodil/section12/lengthKind/ExplicitTests.tdml

```diff
@@ -32,96 +32,96 @@
     </dfdl:defineFormat>
     <dfdl:format ref="ex:GeneralFormat" />
 
-		<xs:element name="address" dfdl:lengthKind="implicit">
-			<xs:complexType>
-				<xs:sequence dfdl:sequenceKind="ordered">
-					<xs:element name="houseNumber" type="xs:string"
-						dfdl:lengthKind="explicit" dfdl:length="6" />
-					<xs:element name="street" type="xs:string"
-						dfdl:lengthKind="explicit" dfdl:length="20" dfdl:ref="trimmed" />
-					<xs:element name="city" type="xs:string"
-						dfdl:lengthKind="explicit" dfdl:length="20" dfdl:ref="trimmed"/>
-					<xs:element name="state" type="xs:string"
-						dfdl:lengthKind="explicit" dfdl:length="2"/>
-				</xs:sequence>
-			</xs:complexType>
-		</xs:element>
-
-	</tdml:defineSchema>
-
-	<tdml:parserTestCase name="Lesson1_lengthKind_explicit"
-		root="address" model="lengthKind_explicit" description="lengthKind='explicit' - DFDL-12-039R"
-		roundTrip="twoPass">
-		<tdml:document><![CDATA[000118Ridgewood Circle    Rochester           NY]]></tdml:document>
-		<tdml:infoset>
-			<tdml:dfdlInfoset>
-				<address>
-					<houseNumber>000118</houseNumber>
-					<street>Ridgewood Circle</street>
-					<city>Rochester</city>
-					<state>NY</state>
-				</address>
-			</tdml:dfdlInfoset>
-		</tdml:infoset>
-	</tdml:parserTestCase>
-
-	<tdml:defineSchema name="test_ExplicitLengthBits" elementFormDefault="unqualified">
+    <xs:element name="address" dfdl:lengthKind="implicit">
```

Comment:
Generally it is better to do these sort of whitespace-only improvements which have no actual code change involved, in a separate commit at least, and perhaps a separate PR. That way we would still be able to see what you changed and added here instead of just seeing "a whole lot of green", and having to gestalt what's modified and what's new.